src/core/config/OperationConfig.js

```diff
@@ -3204,6 +3205,13 @@ const OperationConfig = {
             }
         ]
     },
+    "Micrsoft Script Decoder": {
+        description: "Decodes Microsoft Encoded Script files that have been encoded with Microsoft's custom encoding.",
```

Comment:
Can you add a bit more detail here to describe what kind of script files this works on (Jscript.Encode, VBE etc.)? This is important as users may search for these terms and this operation should be listed for them. It might also be worth including an example in `<code>` tags. The one in your test could be suitable.